cbscript.py

```diff
@@ -75,38 +77,42 @@ def try_to_compile(self):
 		except SyntaxError as e:
 			self.log(str(e) + '\a')
 		except CompileError as e:
-			self.log(str(e) + '\a')
+			pretty_print_error(e)
```

Comment:
`pretty_pint_error` is not adding terminal bell character `\a` to end like existing system